internal/config/config.go

```diff
@@ -39,6 +39,12 @@ type Config struct {
 
 	// Address of Rekor server
 	Rekor string
+	// Rekor storage mode to operate in. One of [online, offline] (default: online)
+	// online - Commit SHAs are stored in Rekor, requiring online verification for all commit objects.
+	// offline - Hashed commit content is stored in Rekor, with Rekor attributes
+	// necessary for offline verification being stored in the commit itself.
+	// Note: online verification will be deprecated in favor of offline in the future.
+	RekorMode string
```

Comment:
Nit: I'd usually make this an enum

Reply:
I'm not against this, but it feels like it would be a good standalone refactor.